package.json

```diff
@@ -1,21 +1,122 @@
 {
-    "private": true,
-    "scripts": {
-        "dev": "npm run development",
-        "development": "mix",
-        "watch": "mix watch",
-        "watch-poll": "mix watch -- --watch-options-poll=1000",
-        "hot": "mix watch --hot",
-        "build": "npm run production",
-        "prod": "npm run production",
-        "production": "mix --production"
+  "name": "mantle",
+  "private": true,
+  "license": "GPL-2.0-or-later",
+  "scripts": {
+    "prebuild": "check-node-version --package",
+    "build": "npm run production",
+    "predev": "check-node-version --package",
+    "dev": "npm run development",
+    "development": "mix",
+    "prehot": "check-node-version --package",
+    "hot": "mix watch --hot",
+    "preprod": "check-node-version --package",
+    "prod": "npm run production",
+    "production": "mix --production",
+    "pretest:watch": "check-node-version --package",
+    "test:watch": "jest --watch",
+    "pretest": "check-node-version --package",
+    "test": "jest",
+    "preupdate-dependencies": "check-node-version --package",
+    "update-dependencies": "node bin/update-dependencies.js",
```

Comment:
If the update dependencies script is swapped in favor or `@wordpress/scripts` you could add the script here. Example in the starter plugin: https://github.com/alleyinteractive/wp-starter-plugin/blob/feature/wp-scripts/package.json#L33-L34